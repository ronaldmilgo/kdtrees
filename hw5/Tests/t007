#!/bin/bash
# kdtree_for_each_range for non-empty subset

trap "/usr/bin/killall -q -u $USER ./Unit 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./Unit ]; then
  echo './Unit is missing or not executable'
  echo './Unit is missing or not executable' 1>&2
  exit 1
fi


/c/cs474/bin/run -stdout=stdout.out -stderr=/dev/null ./Unit 7 < /dev/null
cat stdout.out  | /usr/bin/sort -n
