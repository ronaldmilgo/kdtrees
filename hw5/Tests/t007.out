22.189 -157.489
22.589 -157.028
